src/app/auth/login/login.component.html

```diff
@@ -0,0 +1,28 @@
+<div class="login_container">
+    <div class="login_container__name">Login</div>
+    <form>
+        <p>
```

Comment:
Do we need <p> here and above? If we need some margins, I would add them to mat-form-field firectly